### PR TITLE
BA-338 Add custom giftcards

### PR DESCRIPTION
**Pull request by DjellzeBllaca**

### CODE REVIEW

File: example/transaction/customGiftcard.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        cardNumber: '0000000000000000001',
+        pin: '1000',
+    })
+    .request();
+//Refund
+giftcard
+    .refund({
+        originalTransactionKey: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
+        amountCredit: 10,
+        invoice: 'Custom Giftcard Refund',
```
Comment (vildanbina) [inline example/transaction/customGiftcard.js:20]: can you fix this invoice value to not confuse anyone, as we did in afterpay
`invoice: uniqid()`

File: example/transaction/customGiftcard.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        cardNumber: '0000000000000000001',
+        pin: '1000',
+    })
+    .request();
+//Refund
+giftcard
+    .refund({
+        originalTransactionKey: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
+        amountCredit: 10,
+        invoice: 'Custom Giftcard Refund',
```
Comment (vildanbina) [inline example/transaction/customGiftcard.ts:20]: same here

**Review by vildanbina — CHANGES_REQUESTED** [review]

File: example/transaction/customGiftcard.js (hunk 1 above)
Comment (DjellzeBllaca) [reply example/transaction/customGiftcard.js:21]: Fixed

File: example/transaction/customGiftcard.ts (hunk 2 above)
Comment (DjellzeBllaca) [reply example/transaction/customGiftcard.ts:21]: Done

**Review by vildanbina — APPROVED** [review]

**Pull request merged**